CHANGELOG.md

```diff
@@ -1,3 +1,71 @@
+### citus v5.2.0 (August 15, 2016) ###
+
+* Drops support for PostgreSQL 9.4; PostgreSQL 9.5 is required
+
+* Adds support for schema use in queries and functions
```

Comment:
The 'functions' part was confusing. Maybe something like "Adds Schema support for tables and other named objects like types and operators"?